src/app/student-round.service.ts

```diff
@@ -145,6 +145,53 @@ export class StudentRoundService {
     shareReplay(1),
   );
 
+  interactions$: Observable<Interaction[] | null> =
+  combineLatest([this.sessionService.currentRoundPath$, this.authService.currentUser$]).pipe(
+    switchMap(([path, user]) =>
+      path && user
+        ? this.firebaseService.getStudentInteractions(path, user.uid)
+        : of(null)),
```

Comment:
I think we should return an empty array here, that would be consistent with how we handle other collections and arrays and reduces the need for null checks.